src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -174,7 +174,7 @@ const SignOrExecuteForm = ({
           onFormSubmit={onAdvancedSubmit}
         />
 
-        {(error || (willExecute && gasLimitError)) && (
```

Comment:
You sure about removing `willExecute`? If you checked the Execute box, and there was an error, but then you changed your mind and unchecked it, the error should disappear.

Reply:
Will take a look at this case and revert the change + additional tests if necessary

Reply:
You are right it kept displaying the error. I reverted the change and added an additional test to cover this behaviour.